### DISCUSSION TITLE
When installing via `zplug`:  no such keymap `menuselect`

### DISCUSSION THREAD

**Discussion by jottr** [marlonrichert/zsh-autocomplete#452] [Q&A] [answered]
After installing `zsh-autocomplete` via [`zplug`](https://github.com/zplug/zplug) and restarting the shell I get the following error:

```
.autocomplete.key-binding.precmd:bindkey:46: no such keymap `menuselect'
.autocomplete.key-binding.precmd:bindkey:51: no such keymap `menuselect'
```
I tried to use `defer` to load the plugin after `compinit` has run, but the issue remains:

```
# .zplug.zsh
zplug 'marlonrichert/zsh-autocomplete', defer:2
```
What could be causing this?

**Comment by marlonrichert** [comment, marked as answer]
Looks like [the `menuselect` keymap](https://zsh.sourceforge.io/Doc/Release/Zsh-Modules.html#Menu-selection) (installed by [Zsh’s `zsh/complist` module](https://zsh.sourceforge.io/Doc/Release/Zsh-Modules.html#The-zsh_002fcomplist-Module)) has been deleted, probably by you or a plugin [calling `bindkey -d`](https://zsh.sourceforge.io/Doc/Release/Completion-Widgets.html#Completion-Widgets). Try commenting out lines from your `.zshrc` file until you find where that is done.

Regarding Zplug, see this comment of mine: https://github.com/marlonrichert/zsh-edit/issues/10#issuecomment-1141774661

**Reply by jottr** [reply]
Thanks @marlonrichert. I already planned to move away from `zplug`.
I guess I'll have to do _divide and conquer_ debugging as you suggest.
